docs/content/any/project/changelogs/2022-08-05.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Release 2022-08-05
+menuTitle: 2022-08-05
+any: true
+description: >-
+  Changelog for Release 2022-08-05 (0.26.2) containing new features,
+  bug fixes, and more.
+---
+
+## `oso 0.26.2`
+
+
+We don't have such a thing as "Contributor of the Month", but if we did, it would certainly
+go to [Kevin Kirsche](https://github.com/kkirsche) for submitting PRs:
+
+[1587](https://github.com/osohq/oso/issues/1587), [1586](https://github.com/osohq/oso/issues/1586), [1584](https://github.com/osohq/oso/issues/1584), [1585](https://github.com/osohq/oso/issues/1585), [1581](https://github.com/osohq/oso/issues/1581), [1582](https://github.com/osohq/oso/issues/1582), [1583](https://github.com/osohq/oso/issues/1583), [1555](https://github.com/osohq/oso/issues/1555), [1594](https://github.com/osohq/oso/issues/1594), [1595](https://github.com/osohq/oso/issues/1595), [1604](https://github.com/osohq/oso/issues/1604), [1605](https://github.com/osohq/oso/issues/1605), [1606](https://github.com/osohq/oso/issues/1606). Phew!
+
+
+#### Other bugs & improvements
+
+- Removed an unnecessary write lock that is acquired as part of a query construction,
```

Comment:
nit: Change is to was? "Removed an unnecessary write lock that was acquired as part of a query construction..."